javascript/apps/taiga/src/app/modules/project/feature-kanban/project-feature-kanban.component.ts

```diff
@@ -286,6 +286,29 @@ export class ProjectFeatureKanbanComponent {
         );
       });
 
+    this.wsService
+      .projectEvents<{ workflowStatus: WorkflowStatus }>(
+        'workflowstatuses.update'
+      )
+      .pipe(untilDestroyed(this))
+      .subscribe((eventResponse) => {
+        const workflowStatusResponse =
+          eventResponse.event.content.workflowStatus;
+        const status = {
+          id: workflowStatusResponse.id,
```

Comment:
maybe you can use pick.ts?

Reply:
No sabía usar pick, creo que está bien. He aprovechado para refactorizar otro evento igual que yo básicamente me había fusilated